nullaway/src/test/java/com/uber/nullaway/NullAwayTryConstructsTests.java

```diff
@@ -0,0 +1,83 @@
+package com.uber.nullaway;
+
+import org.junit.Test;
+
+public class NullAwayTryConstructsTests extends NullAwayTestsBase {
```

Comment:
I wonder if these shouldn't be just considered core tests, since they test support of basic Java language constructs without models/handlers/configuration/etc?

Reply:
Yes, that's true.  @Gellert5225 can you put these tests back into the core tests and get rid of this file?